metron-deployment/vagrant/Kerberos-setup.md

```diff
@@ -263,5 +272,12 @@ cat sample-yaf.txt | ${HDP_HOME}/kafka-broker/bin/kafka-console-producer.sh --br
 ${HDP_HOME}/kafka-broker/bin/kafka-console-consumer.sh --zookeeper ${ZOOKEEPER}:2181 --security-protocol PLAINTEXTSASL --topic yaf
 ```
 
+##### Modify the sensor-stubs to send logs via SASL
+```
+sed -i 's/node1:6667 --topic/node1:6667 --security-protocol PLAINTEXTSASL --topic/' /opt/sensor-stubs/bin/start-*-stub
+# Restart the appropriate sensor-stubs
+for sensorstub in bro snort; do service sensor-stubs stop $sensorstub; service sensor-stubs start $sensorstub; done
```

Comment:
I'm still seeing an issue with `service sensor-stubs restart bro snort` - it looks like it's restarting all `sensor-stubs`, including `yaf`, even when I pass specific sensor stubs.  Not a big issue functionally, but this was why I went back to the for loop.  Should probably just fix the `service sensor-stubs` via a separate JIRA.  Agree?
```
[root@node1 ~]# service sensor-stubs restart bro snort
Stopping sensor-stubs...
..       bro: Stopped [510]
       yaf: Not running
..     snort: Stopped [32669]
Starting sensor-stubs...
       bro: Ok [20884]
       yaf: Ok [20889]
     snort: Ok [20912]
[root@node1 ~]# service sensor-stubs status
Checking sensor-stubs...
       bro: Running [20884]
       yaf: Running [20889]
     snort: Running [20912]
```